src/extension/ScriptRunner.ts

```diff
@@ -187,19 +187,126 @@ export class ScriptRunner {
     })
   }
 
-  private postProblem(str: string): void {
-    const dataList = str.split('\n')
-    const diagnosticDataList = dataList[2].split(':')
-    const rowPosition = parseInt(diagnosticDataList[1]) - 1
-    const colStartPosition = parseInt(diagnosticDataList[2]) - 1
-    const startPosition = new Position(rowPosition, colStartPosition)
-    const colEndPosition = colStartPosition + dataList[3].length - 1
-    const endPosition = new Position(rowPosition, colEndPosition)
-    const range = new Range(startPosition, endPosition)
-    const errorMsg = diagnosticDataList[4]
-    const diagnostic = new Diagnostic(range, errorMsg)
-    const diagnostics: Diagnostic[] = [diagnostic]
-    const uri = Uri.parse(diagnosticDataList[0])
-    this.diagnosticCollection.set(uri, diagnostics)
+  private postProblems(confFile: string): void {
+    const ignoreFolderRegex = '{certora-logs,conf}'
+    const resourceErrorsFile = 'resource_errors.json'
+    vscode.workspace
+      .findFiles(resourceErrorsFile, ignoreFolderRegex, 1)
```

Comment:
yes It looks for resouce_errors.json recursively.
I did it like that because I thought maybe the file will not alway appear at the workspace main folder/ maybe the path to it will change with updates to certora tool.